bench/Micro.Benchmark/Benchmarks/Hardware/Compare.cs

```diff
@@ -236,7 +236,7 @@ private static int CompareScalarAltPopCount(void* p1, void* p2, int size)
             //       we can use that we know we are running on little endian hardware and the very first bit set 
             //       will correspond to the first byte which is different. 
 
-            bpx += Popcnt.PopCount((ulong)((long)xor & -(long)xor) - 1) >> 3;
+            bpx += Popcnt.PopCount((uint)((long)xor & -(long)xor) - 1) >> 3;
```

Comment:
Why do we have all these changes?

Reply:
Intrinsics is now a part of a framework and Popcnt.PopCount requires the value to be an uint.